### PR TITLE
Sqlite

### PR DESCRIPTION
**Pull request by chriddyp**
hey @etpinard did i do this right?

### CODE REVIEW

**Comment by etpinard** [comment]
@chriddyp 

Looks like the conversion worked like a charm.

Make the two little modifs, `make run-publish nb=sqlite` , push and merge.

File: _published/includes/sqlite/body.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+       <code>
+        iotools
+       </code>
+       . It's pretty easy to stream a CSV into SQLite and SQLite requires no setup. The SQL query language is pretty intuitive coming from a
+      </p>
+     </div>
+    </div>
+   </div>
+  </div>
+ </div>
```
Comment (chriddyp) [inline _published/includes/sqlite/body.html]: @etpinard ugh looks like it got cutoff

Comment (etpinard) [inline _published/includes/sqlite/body.html]: You mean:

```html
<code>
  iotools
</code>
.
```

Yeah, I know. Good luck trying to fix it though.

Comment (chriddyp) [inline _published/includes/sqlite/body.html]: nah, I mean the notebook is cutoff, only the first section exported to html!

Comment (etpinard) [inline _published/includes/sqlite/body.html]: Oh. Didn't that happen to you before?

Comment (chriddyp) [inline _published/includes/sqlite/body.html]: yeah :(

**Pull request merged**